---
title: 'Comparison of SARS-CoV-2 variants withINSaFLU and galaxyproject/SARS-CoV-2workflows'
tags:
  - COVID-19
  - galaxyproject
  - INSaFLU
  - SARS-CoV-2
  - variant calling
  - workflow
  - Covid-19 Virtual BioHackathon 2020
authors:
  - name: Nazlı Sıla Kara
    affiliation: 1
  - name: Meltem Kutnu
    affiliation: 2
  - name: Yasemin Utkueri
    affiliation: 3
  - name: Funda Yılmaz
    affiliation: 4
  - name: Elif Bozlak
    affiliation: 5,6
  - name: Evrim Fer
    affiliation: 7
affiliations:

  - name: Department of Medical Biochemistry, Faculty of Medicine, Istinye University, Istanbul,Turkey
    index: 1
  - name: Department of Biological Sciences, Middle East Technical University, Ankara, Turkey
    index: 2
  - name: Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, Turkey
    index: 3
  - name: Radboud University, Donders Centre for Cognition, Nijmegen, 6525 HR, The Netherlands
    index: 4
  - name: Institute of Animal Breeding and Genetics, University of Veterinary Medicine Vienna,Vienna, 1210, Austria
    index: 5
  - name: Vienna Graduate School of Population Genetics, Vienna, Austria
    index: 6
  - name: Department of Genetics, University of Arizona, Tucson, USA
    index: 7
date: 19 April 2020
bibliography: paper.bib
---
# Introduction

Covid-19 Virtual BioHackathon 2020 hosts many ongoing projects on workflows about either for the development of the existing ones or combining two or more of them for better functioning.Some of these workflows are Galaxy Project, INSaFLU and nf-core. In these pipelines, the inputs are genome sequence reads mostly coming from Illumina but also Oxford Nanopore, which are analyzed through these workflows by branching whether it is single end or paired end, as output, the annotated variant lists including single nucleotide polymorphisms (SNPs) and small insertion deletion (indel) are provided. We compared published variants by Galaxy project with produced variants by INSaFLU workflow in the course of our study.

# References
